java/java.hints/test/unit/src/org/netbeans/modules/java/hints/errors/CreateMethodTest.java

```diff
@@ -223,7 +222,7 @@ public void test223011b() throws Exception {
                         "        Field f = method(c1, c2);\n" +
                         "    }\n" +
                         "    private <T extends Number & CharSequence, E extends Integer> Field method(Class<E> c1, Class<T> c2) {\n" +
-                        "        throw new UnsupportedOperationException(\"Not supported yet.\"); // Generated from nbfs://nbhost/SystemFileSystem/Templates/Classes/Code/GeneratedMethodBody\n" +
```

Comment:
This is a useful link that I used as I am not satisfied with `new UnsupportedOperationException(\"Not supported yet.\")`.

Reply:
its also editable via Tools -> Templates -> Java/Code Snippets. It should be picked up during a NB version upgrade too as far as i know, so you have edit it only once.

Reply:
I have always known the template is in Tools -> Templates. But I never bothered searching for it! Having the link/action/hint next to the generated text block at the time the incorrect (as perceived by user) block is inserted is essential for people to really invoke the customization.

Reply:
Moreover the `Tools -> Templates` UI isn't in VSCode. Clicking the URL is the only way to customize the template in VSCode.

Reply:
@JaroslavTulach what happens if the user deletes the link in the template then?!

Reply:
> Moreover the `Tools -> Templates` UI isn't in VSCode. Clicking the URL is the only way to customize the template in VSCode.

now I understand the problem. VSCode is missing a feature.

Reply:
@mbien not entirely. The text suggesting that a generated snippet can be customized has been there for a long time. Especially novice users reminded that they can adjust the IDE to his/her own style (i.e. the UnsupportedException default body is seen as good or bad, depending on coding style).

*I opt for leaving* the live links in there. If a presentation does not seem nice (?), we can enhance Folding SPI (now allows only to provide plaintext for the placeholder, could provide i.e. JComponent or TextLayout for the entire fold contents.

Reply:
would be curious what templates you guys are using for the generated methods